src/main/java/pixelperfect/route/AsteroidNode.java

```diff
@@ -0,0 +1,24 @@
+package pixelperfect.route;
+
+/**
+ * An asteroid field Node which belongs in a Route. The asteroid field will be a dangerous location
+ * to fly in.
+ * 
+ * @author Floris Doolaard
+ *
+ */
+public class AsteroidNode extends RouteNode {
+
+  /**
+   * The constructor for the AsteroidNode.
+   * 
+   * @param summary
+   *          , a summary of the AsteroidNode.
+   * @param description
+   *          , a description of the AsteroidNode.
+   */
+  public AsteroidNode(String summary, String description) {
+    super(summary, description);
```

Comment:
Shouldn't those be their own subtype, meaning you will have LargeAsteroidField and SmallAsteroidField (maybe subtypes of AsteroidField). The only reason for having the description parameterized is to allow some variations on the descriptions while the functionality remains exactly the same, is this something we will do?